_i18n/de/resources/moneropedia/pruning.md

```diff
@@ -16,18 +16,16 @@ Pruned-Nodes keine Nachteile bezüglich der Privatsphäre oder Sicherheit.
 
 Es gibt drei Möglichkeiten des Prunings:
 
-- *Synchronisieren eines Pruned-Nodes von Grund auf:* Du synchronisierst und
```

Comment:
example of "fuzzy" not cleaned up by the original commit author/dropping translations. there are other cases. todo